postgres/index.js

```diff
@@ -76,7 +76,7 @@ function setup(testPostgresHost) {
     })
     .then(() => createTables())
     .then(() => ({ server: `${postgresHost}:${POSTGRES_PORT}` }))
-    .catch(logGenericError);
+    .catch(logGenericError(__filename));
```

Comment:
`logGenericError` is a decorator function (ie. It's a function that returns a new function).

Calling it here with `__filename` results in something equivalent to:
```
.catch((err) => log('error', err,message);
```